.github/workflows/ci.yml

```diff
@@ -6,13 +6,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: ["2.5", "2.6", "2.7"]
+        ruby: ["2.5", "2.6", "2.7", "3.0"]
 
     runs-on: macos-latest
     steps:
     # Setup env
     - uses: actions/checkout@v2
-    - uses: actions/setup-ruby@v1
+    - uses: ruby/setup-ruby@v1
```

Comment:
[actions/setup-ruby](https://github.com/actions/setup-ruby) is deprecated and recommended to use [ruby/setup-ruby](https://github.com/ruby/setup-ruby).